Tests/FoundationTests/MeasurementExtensionsTests.swift

```diff
@@ -0,0 +1,48 @@
+// MeasurementExtensionsTests.swift - Copyright 2020 SwifterSwift
+
+@testable import SwifterSwift
+import XCTest
+
+#if canImport(Foundation)
+import Foundation
+
+@available(OSX 10.12, tvOS 10.0, watchOS 3.0, *)
+class MeasurementExtensionsTests: XCTestCase {
+    func testDegree() {
+        let angle = Measurement.degree(2.28)
```

Comment:
Use a top-level constant for the `value`, no need to keep writing 2.28 everywhere.

Reply:
Thanks! I've made some modifications.